tests/variable/variable.cpp

```diff
@@ -1700,6 +1700,115 @@ TEST(Variable, variableInterfaceDontDowngrade)
     EXPECT_EQ("public", v4->interfaceType());
 }
 
+TEST(Variable, connectionsPersistAfterImporting)
+{
+    auto model = libcellml::Model::create("model");
+    auto importedComponent = libcellml::Component::create("importedComponent");
+    model->addComponent(importedComponent);
+
+    auto importer = libcellml::ImportSource::create();
+    importer->setUrl("importedModelWithMaps.cellml");
+    importedComponent->setImportSource(importer);
+    importedComponent->setImportReference("importMe");
+
+    EXPECT_TRUE(model->hasUnresolvedImports());
+    model->resolveImports(resourcePath());
+    EXPECT_FALSE(model->hasUnresolvedImports());
+
+    model->flatten();
+    EXPECT_NE(nullptr, model->component("importedComponent"));
+    EXPECT_NE(nullptr, model->component("importedComponent")->component("child1"));
+    EXPECT_NE(nullptr, model->component("importedComponent")->component("child2"));
+    EXPECT_NE(nullptr, model->component("importedComponent")->component("child1")->variable("x"));
+    EXPECT_NE(nullptr, model->component("importedComponent")->component("child2")->variable("x"));
+
+    auto x1 = model->component("importedComponent")->component("child1")->variable("x");
+    auto x2 = model->component("importedComponent")->component("child2")->variable("x");
+    EXPECT_EQ(size_t(1), x1->equivalentVariableCount());
+    EXPECT_EQ(size_t(1), x2->equivalentVariableCount());
+    EXPECT_EQ(x1, x2->equivalentVariable(0));
+    EXPECT_EQ(x2, x1->equivalentVariable(0));
+}
+
+TEST(Variable, connectionsAsymmetricWithRepeatedUnits)
```

Comment:
I don't understand what this test is doing.  If I remove it I don't fail any tests nor reduce coverage below 100%.

Reply:
1) Removing a test kinda means that it's not gonna fail ... ;) 
2) Test coverage should test all the behaviours, not just all the lines of code, no? 100% coverage should be the bare minimum ...
3) It's been so long I'm struggling to remember why they were written.  I suspect it was to do with mismatched units causing an equivalence not to be added, hence the two tests where one (this one) has valid units between the connected variables, and the other where they're not the same (ms and mm).  This PR came through a maze of issues though, so I'll delete the tests as I'm pretty sure the behaviour is covered elsewhere, and the changes to the `Importer` class will overwrite much of this in any case.  8114ded1